### DISCUSSION TITLE
Variables in CSV URLs

### DISCUSSION THREAD

**Discussion by thamps** [grafana/grafana-infinity-datasource#150] [Q&A] [answered]
Hi,

I am trying to understand if there is a way to use Grafana Dashboard Variable in the URL.

ex. http://someplace/{{ environment }}/reports.csv

Thanks
A

**Comment by yesoreyeram** [comment, marked as answer]
yes.
![image](https://user-images.githubusercontent.com/153843/133039229-6d828bfa-7f2f-4c45-b9aa-6833ef232fe6.png)

**Reply by yesoreyeram** [reply]
@thamps - did this solve the issue? If yes, mark as answer.

**Reply by thamps** [reply]
Oh yes, thanks for the solution. Cheers!
